lti_emailer/settings/local.py

```diff
@@ -15,3 +15,14 @@
 }
 
 dictConfig(LOGGING)
+
+CANVAS_URL = 'http://canvas.icommons.harvard.edu/'
```

Comment:
`secure.py` seems to largely be a misnomer by now.  `secure_and_environment_specific.py` would be a better title.  we've been using it to store settings that vary by environment as well as actual security-sensitive values.